stats/lib/handler.js

```diff
@@ -51,29 +61,24 @@ const enableCors = (req, res) => {
 const handler = async (req, res, pgPools) => {
   // Caveat! `new URL('//foo', 'http://127.0.0.1')` would produce "http://foo/" - not what we want!
   const { pathname, searchParams } = new URL(`http://127.0.0.1${req.url}`)
-  const segs = pathname.split('/').filter(Boolean)
```

Comment:
Please preserve this. It's handling edge cases like `//` in the path or a path ending with a trailing slash.

For example:

```js
const endpoint = '/' + pathname.split('/').filter(Boolean).join('/')
if (req.method === 'GET' && endpoint === '/deals/daily') {
// etc.
```

We can build on top of this a bit more, but I am not sure if that's not too much?

```js
const endpoint = req.method + ' /' + pathname.split('/').filter(Boolean).join('/')
if (endpoint === 'GET /deals/daily') {
```

Reply:
No tests failed so I assumed this was fine. I remember having discussion about this, so I changed it back - albeit in a different way. What do you think?

Reply:
I like the new way 👍🏻